L1Trigger/TrackerTFP/interface/KalmanFilterFormats.h

```diff
@@ -21,7 +21,47 @@ enabling tuning of bit widths
 
 namespace trackerTFP {
 
-  enum class VariableKF { begin, x0 = begin, x1, x2, x3, H00, H12, m0, m1, v0, v1, r0, r1, S00, S01, S12, S13, K00, K10, K21, K31, R00, R11, R00Rough, R11Rough, invR00Approx, invR11Approx, invR00Cor, invR11Cor, invR00, invR11, C00, C01, C11, C22, C23, C33, end, x };
+  enum class VariableKF {
```

Comment:
This is a very ugly clang change, but assume we have to accept it.

Reply:
and that is not even the ugliest ....